plugins/metal/plugin.go

```diff
@@ -38,7 +40,23 @@ var Plugin = plugins.Plugin{
 // map MAC address to inventory name
 var inventoryMap map[string]string
 
-// args[0] = path to configuration file
+// default inventory name prefix
+const defaultNamePrefix = "compute-"
+
+type OnBoardingStrategy int
+
+const (
+	OnboardingFromStaticList      = 1
```

Comment:
Do we really need those flags or should we rely on what is set in the configuration file?

Reply:
It is an internal flag, we do rely on what is set in the configuration file:
https://github.com/ironcore-dev/FeDHCP/blob/feature/new-onboarding-strategy/plugins/metal/plugin.go#L100